### PR TITLE
Initial config

### PR DESCRIPTION
**Pull request by mikealfare**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/decision.yml
Comment (emmyoop) [inline .github/ISSUE_TEMPLATE/decision.yml:1]: Is the idea that we would open an issue to drive public discussions and then open a PR from the outcome of the discussions?

**Review by emmyoop — CHANGES_REQUESTED** [review]
It would be nice to have some sort of table of contents.  Similar to how the OSPO does it [here](https://github.com/todogroup/outbound-oss/blob/main/content/00-toc.md).

File: .github/ISSUE_TEMPLATE/decision.yml
Comment (mikealfare) [reply .github/ISSUE_TEMPLATE/decision.yml:1]: I think so. That felt the most natural to me and was quick to set up. But I see this as a process that we will incrementally improve and I was only looking for a starting point. So please say so if you have a different approach you'd like to try.

**Comment by mikealfare** [comment]
> It would be nice to have some sort of table of contents. Similar to how the OSPO does it [here](https://github.com/todogroup/outbound-oss/blob/main/content/00-toc.md).

I added the most basic version of this. We can always add content to it, and the TOC "generation" is currently manual.

**Review by emmyoop — APPROVED** [review]

**Pull request merged**